views/js/submitScore.js

```diff
@@ -0,0 +1,18 @@
+var scorePopUp = function () {
+  var name = prompt("Please enter your name:", "anonymous");
+  var score = getScore();
+  console.log(score);
+  const xhr = new XMLHttpRequest();
```

Comment:
Consider using `fetch()` over `XMLHttpRequest`:

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch

Reply:
Done in a353ae6